designer/client/components/Visualisation/Lines.tsx

```diff
@@ -16,51 +17,59 @@ type State = {
 };
 
 export class Lines extends React.Component<Props, State> {
+  static contextType = DataContext;
```

Comment:
never used?

Reply:
It is, you need to define contextType for class components so you can use this.context